app/controllers/taxon_names_controller.rb

```diff
@@ -18,6 +18,19 @@ def index
     end
   end
 
+  # GET /api/v1/taxon_names
```

Comment:
@LocoDelAssembly Do you anticipate this collapsing into the default `show` and `index` when we get it pollished?

Reply:
@mjy not so sure at this point. The API must have some stability whereas the regular index/show which are used internally for tasks, etc. might not as it could be adapted to better suit our internal needs and potentially break the API spec.

Reply:
@LocoDelAssembly I think it's important that they converge and be the same.  Utlimately we balance the API to serve application development and general JSON requests.  If the API does not serve applications then it's unlikely to be useful for general calls.  I.e. I anticipate that service calls will be more useful than RESTful calls to many folks doing analyses.  Let's merge to experiment, but let's try get these sunk back together as soon as possible.  @jlpereira and I can work with the optimizations in respect to the interface changes required.  It's better todo this sooner than later, otherwise we're managing two (3?) different ways of exposing the data.